.github/workflows/deploy-docker.yml

```
@@ -166,6 +166,10 @@ jobs:
             echo "DB_HOST_SELF_TEAM_ALLOCATION=${{ vars.DB_HOST_SELF_TEAM_ALLOCATION }}" >> .env.prod
             echo "DB_PORT_SELF_TEAM_ALLOCATION=${{ vars.DB_PORT_SELF_TEAM_ALLOCATION }}" >> .env.prod
 
+            echo "APPLE_ISSUER_ID=${{ vars.APPLE_ISSUER_ID }}" >> .env.prod  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)